README.md

```diff
@@ -39,42 +39,42 @@ We use Python for data science, leveraging a number of powerful libraries for da
    ```sh
    $ git clone git@github.com:Journaly/journaly.git
    ```
-1. In your backend directory, locate your `.env.example` file and create a new one alongside it called `.env`
-1. Run `npm ci` in both the `frontend` and `backend` directories
+1. In your backend directory, locate your `.env.example` file and copy the contents into a new `.env` file alongside it.
+1. Run `npm ci` in both the `frontend` and `backend` directories.
 
 #### Setting Up Your Local DB Instance
 
 1. Install Postgres with Homebrew
    _Note that this set your Postgres DB to run when your computer starts up and will stay running in the background_.
 
-```bash
-$ brew install postgres
-$ brew services start postgresql
-```
+   ```bash
+   $ brew install postgres
+   $ brew services start postgresql
+   ```
 
-2. Start up your Postgres shell and create your user
+1. Start up your Postgres shell and create your user
```

Comment:
Little markdown trick so you can reorder and add bullet points without changing the numbers around. It will still increment when it's rendered